Doberman/Node.py

```diff
@@ -236,8 +246,12 @@ def merge_field(self, field, packages):
     def process(self, packages):
         new_package = {}
         common_keys = set(packages[0].keys())
+        uncommon_keys = set(packages[0].keys())
+        self.logger.debug(f'Package: {packages[0]}')
         for p in packages[1:]:
+            self.logger.debug(f'Package: {p}')
```

Comment:
I think we can do away with these two log statements as well, now that we fixed the problem they were added to help trace.

Reply:
Done